spec/models/status_message_spec.rb

```diff
@@ -41,22 +41,22 @@
 
       describe ".tag_steam" do
```

Comment:
The tests changes you're making here are :ok:

But I don't see any tests for the new functionality you're adding here.

I see your other pull requests add tests so I assume you just didn't want to invest time in the tests because you weren't sure it will get merged soon?